Cargo.toml

```diff
@@ -16,11 +16,11 @@ name = "simplicity"
 path = "src/lib.rs"
 
 [dependencies]
-bitcoin = { version = "0.29.2", optional = true }
-bitcoin_hashes = "0.11"
+bitcoin = { version = "0.30.0", optional = true }
+bitcoin_hashes = "0.12"
 byteorder = "1.3"
-elements = { version = "0.21.1", optional = true }
-elements-miniscript = { git = "https://github.com/apoelstra/elements-miniscript", tag = "2023-07--rust-simplicity-patch" }
+elements = { version = "0.22.0", optional = true }
+elements-miniscript = "0.2.0"
```

Comment:
In 66c07d2ce8474e20b9bc66a566f654c6b8c5f3b4:

Heads up the commit message still says "patched version of elements-miniscript". But we are actually using the released version.